### PR TITLE
Bug/wdp190703 39/section promotion improve rwd

### PR DESCRIPTION
**Pull request by MiroslawChmielecki**
I improved display section PROMOTION in different sizes and improved just a little display section GALLERY

### CODE REVIEW

File: src/partials/60_section-gallery.html [hunk 1]
```diff
@@ -63,28 +89,28 @@ <h5>Aenean Ru Bristique</h5>
...
-              <img src="images/redChairNice.png" alt="Awesome red chair" />
+              <img src="images/redChairNice.png" alt="redChairNice" />
             </div>
             <div class="gallery-slide-box">
-              <img src="images/blackChair2.png" alt="Elegant black ball chair" />
+              <img src="images/blackChair2.png" alt="blackChair" />
             </div>
             <div class="gallery-slide-box">
-              <img src="images/chairBlue.png" alt="Nice blue chair" />
+              <img src="images/chairBlue.png" alt="chairBlue" />
```
Comment (matwjablonski) [inline src/partials/60_section-gallery.html]: fix alt's

File: src/sass/components/_section--promotion.scss [hunk 2]
```diff
@@ -150,6 +215,74 @@
...
+      @include media-breakpoint-only(xs) {
+        width: 45%;
+        padding: 0;
+      }
+
+      @media (max-width: 430px) {
+        min-height: 40%;
+        width: 100%;
+        background-size: 210px;
+        background-position: 10px 0px;
```
Comment (matwjablonski) [inline src/sass/components/_section--promotion.scss]: `px`

File: src/partials/60_section-gallery.html (hunk 1 above)
Comment (MiroslawChmielecki) [reply src/partials/60_section-gallery.html]: fixed

File: src/sass/components/_section--promotion.scss (hunk 2 above)
Comment (MiroslawChmielecki) [reply src/sass/components/_section--promotion.scss]: deleted unneccessary px

**Pull request merged**